src/main/java/org/kuali/kra/negotiations/lookup/NegotiationDaoOjb.java

```diff
@@ -142,9 +146,69 @@
                 result = new ArrayList<Negotiation>();
             }
         }
+        if (result != null && !result.isEmpty())
```

Comment:
Since all the other filters work correctly, for awards, proposals etc we don't need to filter the entire array of negotiation results, just the ones linked to subaward. I'd like to move the filter inside the getNegotiationsLinkedToSubaward method